packages/common/components/elements/attribution/index.marko

```diff
@@ -0,0 +1,34 @@
+import { getAsObject } from "@parameter1/base-cms-object-path";
```

Comment:
Is there a reason this component is being created within a he site repo as opposed to using the one that exits in the core theme?

Reply:
The one in the existing core theme doesn't support showing both people's names and title in line alongside each other. The core theme has either: 1. The name on one line as a large text tag and the title below it as some sort of smaller text tag than the tag above it or 2. Just the name on one line as a smaller text tag similar in size to the title in the other option. 

Example Option 1: 
<img width="1159" alt="Screen Shot 2021-05-18 at 7 49 42 AM" src="https://user-images.githubusercontent.com/46794001/118653928-a6c3b100-b7ad-11eb-9521-96155cd0692f.png">

Example Option 2:
<img width="1919" alt="Screen Shot 2021-05-18 at 7 49 34 AM" src="https://user-images.githubusercontent.com/46794001/118653972-b216dc80-b7ad-11eb-9a71-fc0d21f1bab4.png">


The blocks either of these use isn't set up in the desired fashion that was requested by the ticket hence why it is being created with the site repo.

Reply:
@zarathustra323  We could ultimately just at the title call into the existing element.  It would only display if the fragment using it returned it. We could also add a displayTitle/displayAuthorTitle param to pass threw to account for it. I had him pull this into the PMMI sites only to avoid that, but can help add it to care instead.

We would then just have to adjust the css to account for the combination of multiple authors with or without titles. You would have to do that either way.  

Thoughts/Preference?

Reply:
@B77Mills I can go either way with it. I'd say it's fine to leave in the site repo for now. If another group also wants to add titles to the attribution component we can move it then.